main.py

```diff
@@ -0,0 +1,48 @@
+import random
+import cell
+import bear
+import constants
+import field
+from field import out_file
+
+
+def is_one_bear_left():
+    live_bears = 0
+    for single_bear in field.list_of_bears:
+        if single_bear.life_count > 0:
+            live_bears += 1
+
+    return live_bears == 1
+
+
+
+def main_algorythm():
+    my_world_field = field.Field()
+
+    whose_turn = 0
+    my_world_field.draw_field()
+
+    field.draw_bears_data()
+
+    while not is_one_bear_left():
+        if field.list_of_bears[whose_turn].life_count > 0:
+            field.list_of_bears[whose_turn].move(my_world_field)
+
+        if whose_turn >= constants.BEARS_AMOUNT - 1:
+            whose_turn = 0
```

Comment:
Instead of using an index, you could do something like this
```python
while not is_one_bear_left()
    for bear in field.list_of_bears:
        ...
```